crates/typst-library/src/compute/calc.rs

```diff
@@ -443,11 +443,13 @@ pub fn fact(
     Ok(fact_impl(1, number).ok_or("the result is too large")?)
 }
 
-/// Calculates a permutation.
+/// Calculates the `k`-permutation of `n`.
```

Comment:
Could you remove the backticks? The first line should always be plain text as it's used as a short description in places where Markdown isn't expected.